test/system/system_test.go

```diff
@@ -548,6 +547,58 @@ var _ = Describe("Kratix", func() {
 				Eventually(platform.kubectl("get", "promise")).ShouldNot(ContainSubstring(bashPromiseName))
 			})
 		})
+
+		When("creating multiple resource requests", func() {
+			BeforeEach(func() {
+				platform.eventuallyKubectl("apply", "-f", cat(bashPromise))
+				platform.eventuallyKubectl("get", "crd", crd.Name)
+				worker.eventuallyKubectl("get", "namespace", declarativeWorkerNamespace)
+				platform.kubectl("create", "ns", "test")
+			})
+
+			AfterEach(func() {
+				platform.kubectl("delete", "ns", "test")
+			})
+
+			It("works", func() {
```

Comment:
```suggestion
			It("creates uniques RBAC resources for both requests, including cross-namespaces resources", func() {
```

something like that?